Andy Solar System Test/Assets/rotate.cs

```diff
@@ -5,16 +5,20 @@
 public class rotate : MonoBehaviour {
 
 	public float rotateSpeed = -5.0f;
-
+	private float revolutionSpeed = 1.0f;
+	public void SetRevolution(float revolutionSpeed)
+	{
+		this.revolutionSpeed = revolutionSpeed;
+	}
 	// Use this for initialization
 	void Start () {
-		
+
 	}
-	
+
 	// Update is called once per frame
 	void Update () {
 
-		transform.RotateAround (Vector3.up, rotateSpeed * Time.deltaTime);
+		transform.Rotate (Vector3.up, rotateSpeed* revolutionSpeed * Time.deltaTime);
```

Comment:
This is probably right, but I don't understand the change.  Can you explain please?

Reply:
Rotate around is deprecated. It didn't work correctly when I changed it.